src/navigation/appNavigation.js

```diff
@@ -64,6 +64,7 @@ import BackupPhraseScreen from 'screens/BackupPhrase';
 import BackupPhraseValidateScreen from 'screens/BackupPhraseValidate';
 import CollectibleScreen from 'screens/Collectible';
 import BadgeScreen from 'screens/Badge';
+import OTPScreen from 'screens/OTP/OTP';
```

Comment:
```suggestion
import OTPScreen from 'screens/OTP';
```

Reply:
aaa that's why you have `package.json`
- [ ] remove OTP